### PR TITLE
[RFR] Updating pr_check.sh file

### PR DESCRIPTION
**Pull request by mayurilahane**
Ref - https://issues.redhat.com/browse/RHIROS-494

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+export IQE_CJI_TIMEOUT="30m"
+
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+
+source $CICD_ROOT/build.sh
+
+# Deploy patchman to an ephemeral namespace for testing
```
Comment (apuntamb) [inline pr_check.sh:24]:
```suggestion
# Deploy ros to an ephemeral namespace for testing
```

File: pr_check.sh [hunk 2]
```diff
@@ -1,4 +1,28 @@
...
+source $CICD_ROOT/deploy_ephemeral_env.sh
+
+# Run iqe-patchman somke tests with ClowdJobInvocation
```
Comment (apuntamb) [inline pr_check.sh:27]:
```suggestion
# Run iqe-ros somke tests with ClowdJobInvocation
```

**Review by apuntamb — CHANGES_REQUESTED** [review]
Looks good, thanks @mayurilahane ! Did my first round of review and added comments inline.

File: pr_check.sh [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
-pipenv install -d
-flake8
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+export APP_NAME="ros"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="ros"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+export IMAGE="quay.io/repository/cloudservices/ros-backend"
+export DOCKERFILE="Dockerfile"
+export COMPONENTS_W_RESOURCES="vmaas"
```
Comment (digitronik) [inline pr_check.sh:10]: is `vmass` required for ros deployment?

**Review by digitronik — COMMENTED** [review]
LGTM
You need to check vmass is our dependency or not.

File: pr_check.sh (hunk 3 above)
Comment (r14chandra) [reply pr_check.sh:10]: @digitronik we don't require that

Comment (mayurilahane) [reply pr_check.sh:10]: so i guess we can remove 
`export COMPONENTS_W_RESOURCES="vmaas"`

Comment (mayurilahane) [reply pr_check.sh:10]: @apuntamb FYI

File: pr_check.sh (hunk 1 above)
Comment (apuntamb) [reply pr_check.sh:23]: @mayurilahane I can still see `patchman` instead of `ros`. Can you please fix this?

File: pr_check.sh (hunk 2 above)
Comment (apuntamb) [reply pr_check.sh:26]: ditto

File: pr_check.sh [hunk 4]
```diff
@@ -1,4 +1,27 @@
-set -exv
+#!/bin/bash
 
-pipenv install -d
-flake8
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+export APP_NAME="ros"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="ros"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
```
Comment (apuntamb) [inline pr_check.sh:7]: If it wants the `resourceTemplate` name from deploy file, it should be:
```suggestion
export COMPONENT_NAME="ros-backend"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
```

File: pr_check.sh [hunk 5]
```diff
@@ -1,4 +1,27 @@
...
+#!/bin/bash
 
-pipenv install -d
-flake8
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+export APP_NAME="ros"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="ros-backend"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+export IMAGE="quay.io/repository/cloudservices/ros-backend"
```
Comment (apuntamb) [inline pr_check.sh:8]:
```suggestion
export IMAGE="quay.io/cloudservices/ros-backend"
```
Please refer to https://github.com/RedHatInsights/patchman-engine/blob/master/pr_check.sh#L8

File: pr_check.sh (hunk 1 above)
Comment (mayurilahane) [reply pr_check.sh]: Hey my bad,
i committed local changes after that so it made this undo !
Thanks.

**Review by apuntamb — APPROVED** [review]
LGTM! :+1:

**Comment by apuntamb** [comment]
Merged, thanks @mayurilahane for making the changes and @digitronik for the review.

**Pull request merged**